core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -192,17 +190,29 @@ export class ContributorDashboardPageComponent
       if (userInfo.isLoggedIn()) {
         this.userIsLoggedIn = true;
         this.username = userInfo.getUsername();
+        if (this.username !== null) {
+          [this.profilePicturePngDataUrl, this.profilePictureWebpDataUrl] = (
+            this.userService.getProfileImageDataUrl(this.username));
+        } else {
+          this.profilePictureWebpDataUrl = (
+            this.urlInterpolationService.getStaticImageUrl(
+              AppConstants.DEFAULT_PROFILE_IMAGE_WEBP_PATH));
+          this.profilePicturePngDataUrl = (
+            this.urlInterpolationService.getStaticImageUrl(
+              AppConstants.DEFAULT_PROFILE_IMAGE_PNG_PATH));
```

Comment:
Can we move these lines to above line 190 since they are the default.

Reply:
Done